chia/harvester/harvester.py

```diff
@@ -117,6 +120,15 @@ async def refresh_plots(self):
         if changed:
             self._state_changed("plots")
 
+    async def _periodically_refresh_plots_task(self):
+        while True:
+
+            # Don't run plot refresh until farmer handshake was done
+            if self.farmer_public_keys and self.pool_public_keys:
```

Comment:
If we start the refresh before handshake is done, we might wait a long time to load the plots. We should load the plots immediately after the handshake. Can we maybe have another loop before this while True?

Reply:
Hmm still some odd behavior going on with the CI tests